server/config.go

```diff
@@ -121,3 +181,57 @@ func (s *Service) ReplaceConfigSection(w http.ResponseWriter, r *http.Request) {
 
 	encodeJSON(w, http.StatusOK, res, s.Logger)
 }
+
+func validLogViewerUIConfig(cfg chronograf.LogViewerUIConfig) error {
```

Comment:
would be good here to have an english description of what a valid `LogViewerUIConfig` is.

Reply:
Good catch, I'll write a comment